src/utils/constants.ts

```diff
@@ -3,6 +3,10 @@ export const supportedWallets = [
     name: "Brave",
     url: "https://brave.com",
   },
+  {
+    name: "Frontier",
+    url: "https://www.frontier.xyz/download",
+  },
```

Comment:
Please shift this entry to the end of the list.

Reply:
ok